build.gradle

```diff
@@ -44,25 +46,12 @@ buildscript {
                 password property("eu.xenit.artifactory.password")
             }
         }
-        // Dynamic Extensions
-        maven {
-            url 'https://oss.sonatype.org/content/repositories/snapshot/'
-        }
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
-        classpath 'org.apache.httpcomponents:httpclient:4.5.3'
-        classpath 'org.apache.httpcomponents:httpmime:4.5.3'
-        classpath 'org._10ne.gradle:rest-gradle-plugin:0.4.2'
-        classpath 'org.codehaus.groovy.modules.http-builder:http-builder:0.7'
-        classpath 'eu.xenit.gradle:alfresco-docker-plugin:4.0.2'
```

Comment:
I suppose those buildscript dependencies are now included in the plugins?

Reply:
I have no idea, except that I tested it and everything works